test/verify/check-packagekit

```diff
@@ -45,8 +45,8 @@ class NoSubManCase(PackageCase):
         # Disable Subscription Manager on RHEL for these tests; subscriptions are tested in a separate class
         # On other OSes (Fedora/CentOS) we expect sub-man to be disabled in yum, so it should not get in the way there
         if self.machine.image in ["rhel-8-2", "rhel-8-2-distropkg", "centos-8-stream"]:
```

Comment:
Fairly sure that we now need to add rhel-8-3 here. Or rather, `.startswith("rhel-")` and same for centos?

Reply:
[Yep, we do](https://logs.cockpit-project.org/logs/pull-13938-20200424-055917-3817512b-rhel-8-3/log.html)

Reply:
Note that this got changed just recently in fe1b8c6277a285990e45b477b2ef654f84e37f95

Reply:
I have reverted part of fe1b8c6

Reply:
Ah yes, that was a temporary hack to work around rhsmd not existing any more in 8.3. Thanks!